server/models/comment.js

```diff
@@ -1,12 +1,9 @@
 const { Schema, model } = require('mongoose');
+
 const dateFormat = require('../utils/dateFormat');
 
 const commentSchema = new Schema(
 	{
-		commentId: {
```

Comment:
I think this is still needed but I dont know with nested comments so I'll look in to it